<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maps</class>
 <widget class="QDialog" name="Maps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>139</width>
    <height>291</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Карты (С полезными метками)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/sZOInfo.png</normaloff>img/sZOInfo.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>99999</width>
     <height>999999</height>
    </size>
   </property>
   <property name="title">
    <string>Карты (С полезными метками)</string>
   </property>
   <widget class="QComboBox" name="OptMaps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Любеч Вокзал от сталкера  Ченик</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Любеч лес</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Везувий</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новая Земля</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Тунгуска</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Тунгусска. Травы (От исследователей КК) и в частности от KirillZVER</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Тунгусска от Rederrr2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Любеч Караван</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Любеч лес (схроны)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ButtomView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtomZoomIn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtomZoomOut">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>zoom:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="ShowGraf">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>798</width>
     <height>651</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>798</width>
     <height>651</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="interactive">
    <bool>true</bool>
   </property>
   <property name="resizeAnchor">
    <enum>QGraphicsView::AnchorViewCenter</enum>
   </property>
  </widget>
  <zorder>ShowGraf</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources>
  <include location="infoco.qrc"/>
 </resources>
 <connections/>
</ui>
